posydon/popsyn/synthetic_population.py

```diff
@@ -515,8 +552,17 @@ def get_dco_merger_rate_density(self, export_cols=None,  working_dir='./', load_
 
         # compute rate density weights
         self.z_rate_density = self.rates.get_centers_redshift_bins()
-        self.rate_density = self.rates.compute_merger_rate_density(w_ijk, z_merger, observable='DCOs', sensitivity=sensitivity)
-        print(f'DCO merger rate density in the local Universe (z={self.z_rate_density[0]:1.2f}): {round(self.rate_density[0],2)} Gpc^-3 yr^-1')
+        self.rate_density = {}
+        total_rate = self.rates.compute_merger_rate_density(w_ijk, z_merger, observable='DCOs', sensitivity=sensitivity)
+        self.rate_density['total'] = total_rate
```

Comment:
Again, define and assign the dictionary value at the same time, this is more efficient in python.